composer.json

```diff
@@ -27,6 +27,7 @@
         "psr/log": "^1.0"
     },
     "require-dev": {
+        "symfony/translation": "^3.4.0||^4.3.0||^5.0.0",
```

Comment:
it's a hard requirement of ValidateBehavior.php.

Reply:
Did you add it to your changes for require so far?

Reply:
Ah I see, I will revert this then.